semantic-source/src/Source/Loc.hs

```diff
@@ -13,6 +13,7 @@ import GHC.Generics (Generic)
 import Prelude hiding (span)
 import Source.Range
 import Source.Span
+import Data.Aeson (ToJSON(..), object, (.=))
```

Comment:
Could you please alphabetize the imports? 🙏

Reply:
yep my bad!